### ISSUE TITLE
Make metrics above installation map on dataverse.org not misleading

### ISSUE THREAD

**Issue by pdurbin** [IQSS/dataverse#5429]
We want the map on dataverse.org to only show the number of installations (35 as of this writing), not "2,872 Dataverses", "52,449 Datasets", "4,102,900 Downloads" as in the screenshot below:

![screen shot 2019-01-03 at 4 24 51 pm](https://user-images.githubusercontent.com/21006/50662453-78463200-0f74-11e9-8a5b-3d4a659061fe.png)

The fix for this should be to comment out a few lines as was already done in https://github.com/IQSS/miniverse/pull/66/files like this:

![screen shot 2019-01-03 at 4 28 14 pm](https://user-images.githubusercontent.com/21006/50662507-a6c40d00-0f74-11e9-9fbc-b96bad624b74.png)

Previously, we were tracking this issue at https://github.com/IQSS/miniverse/issues/65 but @djbrooke said to go ahead and create the issue here in the main issue tracker.

**Comment by pdurbin** [comment]
I believe this issue is blocked by #5497

**Comment by mheppler** [comment]
Dig some digging into the miniverse repo to see if this was a simple HTML edit to comment out some code. It appears that was already done, by @mallove on Jan 12, 2018 2d3116e. That commit is in the `master` branch, but it does not appear to be what is being served to dataverse.org by https://services.dataverse.harvard.edu/miniverse/map/dataverse-org.

I suspect that if the lastest version of the `master` branch for miniverse were to be deployed, the commented out version of the map would correctly show on dataverse.org. Maybe @landreev could look into it as he last worked with the Harvard Library folks to resolve the miniverse db connection issues.

**Comment by landreev** [comment]
Thanks to @mheppler for the explanation - I didn't get that part until now, that the most recent code updates in miniverse were in the master branch, but not in the latest tagged release. 
I'll redeploy from master tomorrow, as we are going to be deploying the 4.11 of Dataverse in production.

**Comment by landreev** [comment]
@mheppler I think this is looking ok now - but please confirm:
![screen shot 2019-02-21 at 2 25 01 pm](https://user-images.githubusercontent.com/737558/53195879-a5ae7400-35e4-11e9-854c-b2e04018104c.png)

**Comment by mheppler** [comment]
Looks good to me. Moving to QA.
